apriltag.h

```diff
@@ -35,6 +35,7 @@ extern "C" {
 
 #include "common/matd.h"
 #include "common/image_u8.h"
+#include "common/image_u8_parallel.h"
```

Comment:
This header is not required in `apriltag.h` as you are not using any of the definitions from `image_u8_parallel.h` in `apriltag.h`. Include `image_u8_parallel.h` where its definitions are actually used - in `apriltag.c`.